src/kbs1.jl

```diff
@@ -0,0 +1,91 @@
+"""
+    test1!(u::Word, v::Word, rs::RewritingSystem, o::Ordering)
+Adds a rule to a rewriting system (if necessary) that insures that there is
+a word derivable form two given words using the rules in rewriting system.
+See [Sims, p. 69].
+"""
+function test1!(u::Word, v::Word, rs::RewritingSystem, o::Ordering)
```

Comment:
the general practice is to place the first argument as the one being modified, i.e.
```suggestion
function test1!(rs::RewritingSystem, u::Word, v::Word, o::Ordering = ordering(rs))
```
I also added the default ordering if we include it into the rewriting system

We open a contest for better name for this function :D my candidate is `deriverule!`